mdoc/mdoc.csproj

```diff
@@ -55,6 +55,8 @@
     <Compile Include="Mono.Documentation\Updater\Formatters\StandardFlagsEnumFormatter.cs" />
     <Compile Include="Mono.Documentation\Updater\Formatters\DefaultAttributeValueFormatter.cs" />
     <Compile Include="Mono.Documentation\Updater\Formatters\AttributeValueFormatter.cs" />
+    <Compile Include="Mono.Documentation\Updater\Formatters\VBFullMemberFormatter.cs" />
+    <Compile Include="Mono.Documentation\Updater\Formatters\VbMemberFormatter.cs" />

```

Comment:
@mpMelnikov just from a naming perspective, can we make sure that the file names follow a standard casing approach, e.g. if we have `VBFullMemberFormatter`, then we also call the adjacent file `VBMemberFormatter`

cc: @joelmartinez

Reply:
@dend Yes, you're right, I'll fix it.

Reply:
Good catch @dend